lib/yookassa/payments.rb

```diff
@@ -31,5 +36,13 @@ def cancel(payment_id:, idempotency_key: SecureRandom.hex(10))
     private
 
     attr_reader :api
+
+    def validate_payment(data)
+      contract = Yookassa::Contracts::PaymentContract.new
+      result = contract.call(data)
+      return if result.success?
+
+      raise CreatePaymentError, result.errors.to_h
```

Comment:
It would be more useful, to have this errors as a hash inside exception instance
CreatePaymentError.new(errors: result.errors)
AFAIR current notation will transform hash to string